yarn-project/ethereum/src/contracts/slashing_proposer.ts

```diff
@@ -142,8 +145,9 @@ export class SlashingProposerContract extends EventEmitter implements IEmpireBas
           data,
         },
         {
-          // Gas estimation is more than 20% off for this tx.
-          gasLimitBufferPercentage: 100,
+          // Gas estimation is way off for this, likely because we are creating the contract/selector to call
+          // for the actual slashing dynamically.
+          gasLimitBufferPercentage: 1000,
```

Comment:
TODO: we can work out if this will end up deploying something, 1000 is wildly high